spec/features/users/03_user_signs_in_spec.rb

```diff
@@ -0,0 +1,59 @@
+require 'rails_helper'
+
+feature "user signs In", %{
+  As unauthenticated user
+  I want to sign in
+  So that I add donut and review the app.
+} do
+  # Acceptance Criteria
+  # [ ] Link to sign in exists in the nav-bar
+  # [ ] Clicking Sign in shows the sign-in form
+  # [ ] Signing in directs us to donuts page
+
+  scenario "user sees sign in" do
+    visit root_path
+    expect(page).to have_link("Sign In")
+  end
+
+  scenario "user sucessfully sign-in" do
```

Comment:
This would be more explicit if it were called `user can sign in with valid credentials`. It describes who is doing the action, what they're doing, and what parameters the action is being done with. It also makes a clear distinction between this happy path and the unhappy path test which should be `user cannot sign in with invalid credentials`